kt/api-generator/src/main/kotlin/godot/codegen/extensions/TypedExtensions.kt

```diff
@@ -61,10 +60,10 @@ fun TypedTrait.isTypedArray() = type?.startsWith(GodotTypes.typedArray) ?: false
 fun TypedTrait.getTypeClassName(): ClassTypeNameWrapper {
     val typeNameWrapper = when {
         type.isNullOrEmpty() -> ClassTypeNameWrapper(UNIT)
-        type == "Signal0" -> ClassTypeNameWrapper(ClassName(signalPackage, type!!))
+        type == "Signal0" -> ClassTypeNameWrapper(ClassName(godotCorePackage, type!!))
         type!!.startsWith("Signal") -> {
             this as EnrichedSignal
-            val className = ClassName(signalPackage, type)
```

Comment:
Was ˋsignalPackage` variable deleted ?

Reply:
Yes it was. Callable and Signal have been moved to the same package as the other coretypes. For consistency. They were initially in different packages back in Godot 3, but now they are full fledged Variant types so I moved them to their rightful place with the others.